xml/System/RuntimeTypeHandle.xml

```diff
@@ -330,7 +330,7 @@
 ## Remarks  
  Using this operator to compare two variables of type <xref:System.RuntimeTypeHandle> causes an ambiguous overload resolution error when compiled. Use the <xref:System.RuntimeTypeHandle.Equals%2A> method instead.  
   
- The equivalent method for this operator is <xref:System.RuntimeTypeHandle.Equals%28System.RuntimeTypeHandle%2CSystem.Object%29?displayProperty=fullName>]]></format>
+ The equivalent method for this operator is <xref:System.RuntimeTypeHandle.Equals%28System.RuntimeTypeHandle%29?displayProperty=fullName>.]]></format>
```

Comment:
This file is showing two more instances of ...
```
Warning	Unspecified
api/System.RuntimeTypeHandle.yml
Invalid cross reference "&lt;xref:System.RuntimeTypeHandle.Equals%28System.Object%2CSystem.RuntimeTypeHandle%29?displayProperty=fullName&gt;".
```
Lines 288 and 378: Drop `System.Object` again from the two lines that have it ...

> \<xref:System.RuntimeTypeHandle.Equals%28System.Object.System.RuntimeTypeHandle%29?displayProperty=fullName>

... to ...

> \System.RuntimeTypeHandle.Equals%28System.RuntimeTypeHandle%2CSystem.Object%29?displayProperty=fullName>

... they need periods at the end, too.